tests/keyboard-test/UbuntuKeyboard/Dummy.qml

```diff
@@ -1,5 +0,0 @@
-import QtQuick 2.0
-
-Item {
-    property bool available: true
-}
```

Comment:
Well the C++ code doesn't check for the available currently anywhere. So we can keep the C++ files and remove the Ubuntu ones. Alternatively we can rename them to something else instead if that makes you happier ;)